utils_methods.py

```diff
@@ -36,30 +37,19 @@ def to_python(self, value):
     def get_prep_value(self, value):
         # Implement encryption logic here
         return encrypt(value)
-
-#If you want to decrypt then you can uncomment this and run... in output you will find the decrypted account number 
+'''
+#In output you will find the decrypted account number 
 # Encoded account number
-encoded_account_number = "'MTIzNDU2Nzg5MDk4NzU0Mw=='"
-
-# Decode from base64
-decoded_bytes = base64.b64decode(encoded_account_number)
-
-# Convert bytes to string
-original_account_number = decoded_bytes.decode("utf-8")
-
-print("Decrypted Account Number:", original_account_number)
+#logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
 
-#db_table columns
+'''
 
-companytable ='companies'
-branchestable ='branches'
-branchbankdetails = 'branch_bank_details'
+encoded_account_number = ""
+decoded_bytes = base64.b64decode(encoded_account_number)    # Decode from base6
+original_account_number = decoded_bytes.decode("utf-8") # Convert bytes to string
 
-#db_table masters columns
-countrytable = 'country'
-statetable = 'state'
-citytable = 'city'
-statusestable = 'statuses'
+#print("Decrypted Account Number:", original_account_number)
+logging.info("Decrypted Account Number: %s", original_account_number)
 
```

Comment:
Remove these line numbers, logging or printing the sensitive data is security issue. For testing purpose you can add but before commit you should remove it

Reply:
Okay Thulasi.